client_lib/pulp/client/commands/repo/sync_publish.py

```diff
@@ -25,10 +25,13 @@
                          'missing files and re-download files as necessary '
                          'rather than just downloading files that are known '
                          'to be missing.')
-DESC_FORCE_FULL_FLAG = _('forces to publish from scratch, even if no changes were '
-                         'introduced since last publish')
+DESC_FORCE_FULL_FLAG_PUBLISH = _('forces to publish from scratch, even if no changes were '
+                                 'introduced since last publish')
+DESC_FORCE_FULL_FLAG_SYNC = _('forces to sync from scratch, even if no changes were '
```

Comment:
what about saying: "forces a full re-sync, where all steps are performed"
Because the main point of making the force full re-sync is not to skip any steps, because every plugin has its own sync optimization that is based of the fact that some of sync steps are skipped. Force sync makes sure that we do perform all steps.